server/ai_process.go

```diff
@@ -1037,13 +1039,35 @@ func submitLiveVideoToVideo(ctx context.Context, params aiRequestParams, sess *A
 		if err != nil {
 			return nil, fmt.Errorf("sub url %w", err)
 		}
-		clog.V(common.VERBOSE).Infof(ctx, "pub %s sub %s", pub, sub)
+		control, err := appendHostname(resp.JSON200.ControlUrl)
+		if err != nil {
+			return nil, fmt.Errorf("control pub url - %w", err)
+		}
+		clog.V(common.VERBOSE).Infof(ctx, "pub %s sub %s control %s", pub, sub, control)
 		startTricklePublish(pub, params)
 		startTrickleSubscribe(sub, params)
+		startControlPublish(control)
 	}
 	return resp, nil
 }
 
+// TODO: Remove it and get requests from POST /stream/update
+func startControlPublish(control *url.URL) {
```

Comment:
This file is getting  large so I have been putting the live video stuff into the [ai_live_video.go](https://github.com/livepeer/go-livepeer/blob/master/server/ai_live_video.go)  file, that is where the `startTricklePublish` and `startTrickleSubscribe` stuff lives